parachain/primitives/beacon/src/types.rs

```diff
@@ -386,6 +391,55 @@ pub struct CompactBeaconState {
 	pub block_roots_root: H256,
 }
 
+/// VersionedExecutionPayloadHeader
+#[derive(Encode, Decode, CloneNoBound, PartialEqNoBound, RuntimeDebugNoBound, TypeInfo)]
+#[cfg_attr(
+	feature = "std",
+	derive(Serialize, Deserialize),
+	serde(deny_unknown_fields, bound(serialize = ""), bound(deserialize = ""))
+)]
+#[codec(mel_bound())]
+pub enum VersionedExecutionPayloadHeader {
+	Capella(ExecutionPayloadHeader),
+	Deneb(ExecutionPayloadHeaderDeneb),
+}
```

Comment:
Rather use modules as a namespace, rather than adding `Deneb` as a suffix.

```suggestion
pub enum VersionedExecutionPayloadHeader {
	Capella(capella::ExecutionPayloadHeader),
	Deneb(deneb::ExecutionPayloadHeader),
}
```

Reply:
Reorganized in https://github.com/Snowfork/snowbridge/commit/3580c4de7d6400880f3a4df171ffb42b5000737b